transaction/std/src/memo/gift_code_funding.rs

```diff
@@ -68,14 +102,14 @@ impl GiftCodeFundingMemo {
             .memo_data
             .iter()
             .enumerate()
-            .position(|(i, b)| i >= Self::HASH_DATA_LEN && b == &0u8)
+            .position(|(i, b)| i >= Self::NOTE_OFFSET && b == &0u8)
```

Comment:
Optional/Equivalent: Use `skip()` to skip past the offset, and use `*b` instead of `&0`
```suggestion
            .skip(Self::NOTE_OFFSET)
            .position(|b| *b == 0u8)
```

Reply:
I tested this, it fails.